spec/index.bs

```diff
@@ -1554,15 +1555,15 @@ interface GPUAdapter {
                         <div class=validusage>
                             - The set of values in |descriptor|.{{GPUDeviceDescriptor/nonGuaranteedFeatures}}
                                 must be a subset of those in |adapter|.{{adapter/[[features]]}}.
-
-                            - Each key in |descriptor|.{{GPUDeviceDescriptor/nonGuaranteedLimits}}
-                                must be the name of a member of [=supported limits=].
                         </div>
 
                     1. If any of the following requirements are unmet,
                         [=reject=] |promise| with an {{OperationError}} and stop.
 
                         <div class=validusage>
+                            - Each key in |descriptor|.{{GPUDeviceDescriptor/nonGuaranteedLimits}}
+                                must be the name of a member of [=supported limits=].
```

Comment:
Reclassifying this as an `OperationError` was suggested by @kainino0x in the review of [this Chrome change](https://chromium-review.googlesource.com/c/chromium/src/+/2915661):

> Hm... the spec as I wrote it requires a TypeError here, but it occurs to me an OperationError might be better to match the behavior between "limit not known" and "limit higher than supported", because probably the former can be considered a special case of the latter....